hedera-mirror-graphql/src/main/java/com/hedera/mirror/graphql/controller/AccountController.java

```diff
@@ -62,6 +66,41 @@ Mono<Account> account(@Argument @Valid AccountInput input) {
                     .map(accountMapper::map));
         }
 
+        if (alias != null) {
+            byte[] aliasBytes = decodeBase32(alias);
+            var account = entityService.getByAlias(aliasBytes, EntityType.ACCOUNT)
+                    .map(accountMapper::map);
+            if (account.isEmpty()) {
+                byte[] evmAddressBytes = new byte[20];
+                int aliasByteLength = aliasBytes.length > 20 ? 20 : aliasBytes.length;
+                System.arraycopy(aliasBytes, 0, evmAddressBytes, 20 - aliasByteLength, aliasByteLength);
+                var entity = fromEvmAddress(evmAddressBytes);
+                if (entity != null) {
+                    account = entityService.getByIdAndType(entity, EntityType.ACCOUNT)
+                            .map(accountMapper::map);
+                }
+            }
+
+            return Mono.justOrEmpty(account);
+        }
+
+        if (evmAddress != null) {
+            byte[] evmBytes = decodeEvmAddress(evmAddress);
+            var account = entityService.getByEvmAddress(
+                            evmBytes,
+                            EntityType.ACCOUNT)
+                    .map(accountMapper::map);
+
+            if (account.isEmpty()) {
+                var entity = fromEvmAddress(evmBytes);
+                if (entity != null) {
+                    account = entityService.getByIdAndType(entity, EntityType.ACCOUNT)
+                            .map(accountMapper::map);
+                }
+            }
+            return Mono.justOrEmpty(account);
```

Comment:
Remove the fallback behavior.
```suggestion
            return Mono.justOrEmpty(entityService.getByEvmAddress(evmBytes, EntityType.ACCOUNT)
                            .map(accountMapper::map));
```